test/integration/virtual-rules/index.html

```diff
@@ -21,21 +21,24 @@
 	</head>
 	<body>
 		<div id="mocha"></div>
+		<script src="area-alt.js"></script>
+		<script src="aria-input-field-name.js"></script>
+		<script src="aria-menuitem-name.js"></script>
+		<script src="aria-progressbar-name.js"></script>
+		<script src="aria-toggle-field-name.js"></script>
+		<script src="aria-tooltip-name.js"></script>
```

Comment:
We need to think of a way to make sure we don't forget to do this. Any thoughts?

Reply:
We could generate the file like we do the other test html files. I've not looked into that part of the build but it should be possible.